db/migrate/20180117201300_make_ko_default_tournament_mode.rb

```diff
@@ -0,0 +1,7 @@
+class MakeKoDefaultTournamentMode < ActiveRecord::Migration[5.1]
+  def up
+    change_column :events, :game_mode, :integer, default: 0, null: false
```

Comment:
Um, this can be part of a change method.

Reply:
And this influences all types of events, should you not put this elsewhere? Like, in the model?

Reply:
yes, but then the migration would not be revertable.

is it event possible to make it model side only?

Reply:
Pending discussion with PO

Reply:
And even then, this migration is not reversible, you need to write a corresponding down method

Reply:
Since you are just changing what is default I don't even think you need down and up. But yes in my opinion this should be part of the model, since maybe Rankinglists or Leagues don't need a default game_mode. You can implement this with another hook. This time `before_save`
Also you can validate the presence of a game_mode in the model which is another way of saying `null:false` although I think this was the case before you added this migrations anyways.